consensus/polybft/fsm.go

```diff
@@ -604,6 +703,28 @@ func (f *fsm) verifyCommitEpochTx(commitEpochTx *types.Transaction) error {
 	return errCommitEpochTxNotExpected
 }
 
+// verifyCommitEpochTx creates commit epoch transaction and compares its hash with the one extracted from the block.
+func (f *fsm) verifyFundRewardWalletTx(fundRewardWalletTx *types.Transaction) error {
+	if f.isEndOfEpoch {
+		localFundRewardWalletTx, err := f.createRewardWalletFundTx()
```

Comment:
what if the fund amount is zero and such tx must not be made

Reply:
So, do you mean that if we have a fund transaction that is with value = 0, then we shouldn't even verify and proceed, just skip it?